doc/generated_testcases_v2.yaml

```diff
@@ -0,0 +1,110 @@
+# The following extensions will be created for each .in that's found.
+# Any key starting with a dot will be interpreted as an extension generator.
+# .ans will be generated before all others.
+.ans: /solutions/accepted/sol.py
+.png: /visualizers/asy.py # Note that directly calling a shell executable is not allowed
```

Comment:
I guess you don't mean you can't call a shell script, but that you can't use shell syntax on this line. I.e. you can write
```
.png: /visualizers/my-bash-script-that-calls-other-stuff.sh
```
but you can't write
```
.png: /visualizers/preprocess.py | grep "foobar" | tac
```